test/conftest.py

```diff
@@ -0,0 +1,29 @@
+import asyncio.unix_events
+
+import pytest
+from httpx import AsyncClient
+
+from main.main import app
+from main.services import session
+
+pytest_plugins = ["test.helpers"]
+
+
+@pytest.fixture(scope="session")
+def event_loop() -> any:
+    loop = asyncio.get_event_loop()
```

Comment:
`get_event_loop` is going to be deprecated
Ref: https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.get_event_loop